doc/doxygen/src/roadmap.md

```diff
@@ -9,85 +9,67 @@ The text and items below are tentative, up for discussion, to be updated by regu
+1. more advanced LPM concepts:
   - sleeping for short periods (in cases where it is not feasible to switch to the idle thread and back) -> mitigate active waiting
 
 
 
 # Peripheral drivers
 (contact/steering: [Hauke](https://github.com/haukepetersen))
 
-1. remodeling  of the `periph/i2c.h` interface and subsequent adaption/rewrite of all existing implementations (-> https://github.com/RIOT-OS/RIOT/issues/6577)
-2. cleanup and unification of low-level timer interfaces (`timer`, `rtt`, `rtc`)
-3. introduction of `spi_slave` interface
-4. introduction of `i2c_slave` interface
+1. cleanup and unification of low-level timer interfaces (`timer`, `rtt`, `rtc`)
+1. introduction of `spi_slave` interface
+1. introduction of `i2c_slave` interface
 
 
 
 # Software Updates
 (contact/steering: [Emmanuel](https://github.com/emmanuelsearch))
 
-1. ~~Basic bootloader, flashing and booting one of multiple firmware slots~~ done, see [riotboot](https://github.com/RIOT-OS/RIOT/tree/master/bootloaders/riotboot)
-2. ~~Secure firmware update over CoAP compliant with SUIT draft spec [draft-ietf-suit-manifest-03](https://tools.ietf.org/html/draft-ietf-suit-manifest-03)~~ done, see [SUIT update example](https://github.com/RIOT-OS/RIOT/tree/master/examples/suit_update)
-3. Update SUIT spec support to comply with [draft-ietf-suit-manifest-04](https://tools.ietf.org/html/draft-ietf-suit-manifest-04)
-4. Modularize to provide toolbox supporting other image storing (e.g. support external flash), transport (other than CoAP), crypto (e.g. secure element).
-5. riotboot support for architectures other than Cortex-M
+1. Modularize to provide toolbox supporting other image storing (e.g. support external flash), transport (other than CoAP), crypto (e.g. secure element).
+1. riotboot support for architectures other than Cortex-M
 
 
 
 # Documentation
 (contact/steering: [Emmanuel](https://github.com/emmanuelsearch))
 
-1. ~~publish roadmap~~ done
-2. ~~Introduce RIOT Developer Memos~~ done, see [RDM0](https://github.com/RIOT-OS/RIOT/tree/master/doc/memos)
-3. Write and publish more RDMs
-4. revamp RIOT website (see [Web Revamp Task Force](https://github.com/RIOT-OS/RIOT/wiki/Website-Revamp-Task-Force))
+1. Write and publish more RDMs
 
 
 
 # Low-level Hardware Support
 (contact/steering: [Alex](https://github.com/aabadie))
 
-1. Improved MIPS support
-2. radio support for TI SensorTag
-3. radio support for Silab Thunderboard
-4. ESP32 support
+1. radio support for TI SensorTag
+1. radio support for Silab Thunderboard
 
 
 
 # Testing
 (contact/steering: [Kaspar](https://github.com/kaspar030))
 
-1. ~~automated unit tests with hardware in the loop (SAMR21 plugged on CI server?)~~ (done, Murdock and Philipp)
-2. automated network functionality tests (e.g. RPL + UDP/PING tests through border router, multi-hop) in IoTLAB dev sites?
-3. ~~leverage PiFleet more?~~ done, 1. uses PiFleet
-4. On-board CI testing in IoT-LAB (as it will provide soon the possibility to add custom nodes)
```

Comment:
@MrKevinWeiss and @Miri: Could you take a look at what is done already here? Don't the automated release test cover this as well?

Reply:
If you want my feedback earlier, you need to mention @miri64 ;-). `@miri` got some very angry replies in the past.

And yes, the release test should cover this.

Reply:
> `@miri` got some very angry replies in the past.
Oopsie :blush:

> And yes, the release test should cover this.

Thx :-)